### ISSUE TITLE
Virage 1 softlock

### ISSUE THREAD

**Issue by avionanx** [Legend-of-Dragoon-Modding/Severed-Chains#1230]
encounter 408, combat hud isnt visible, turns dont begin

**Comment by LordMonoxide** [comment]
The new monster texture unpacker code is missing entries for some monsters, such as 310 (the monster causing the crash here)
```
21:12:22.527 [pool-2-thread-4 legend.game.unpacker.Unpacker:176] INFO : File monsters/308/textures/combat loaded (remaining queued: 11)
Exception in thread "pool-2-thread-5" Exception in thread "pool-2-thread-1" java.lang.RuntimeException: Failed to load file monsters/310/textures/combat
	at legend.game.unpacker.Unpacker.loadFile(Unpacker.java:166)
	at legend.game.unpacker.Unpacker.lambda$loadFile$1(Unpacker.java:174)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.nio.file.NoSuchFileException: .\files\monsters\310\textures\combat
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:234)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:379)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:431)
	at java.base/java.nio.file.Files.readAllBytes(Files.java:3268)
	at legend.game.unpacker.Unpacker.loadFile(Unpacker.java:164)
	... 4 more
java.lang.IndexOutOfBoundsException: Read end 8 out of bounds 0
	at legend.game.unpacker.FileData.checkBounds(FileData.java:211)
	at legend.game.unpacker.FileData.readInt(FileData.java:99)
	at legend.game.tim.Tim.getFlags(Tim.java:29)
	at legend.game.tim.Tim.hasClut(Tim.java:37)
	at legend.game.tim.Tim.getImageDataOffset(Tim.java:21)
	at legend.game.tim.Tim.getImageOffset(Tim.java:65)
	at legend.game.tim.Tim.getImageData(Tim.java:69)
	at legend.game.combat.Battle.loadCombatantTim2(Battle.java:2499)
	at legend.game.combat.Battle.loadCombatantTim(Battle.java:2489)
	at legend.game.combat.Battle.lambda$loadEnemyTextures$2(Battle.java:1410)
	at legend.game.unpacker.Unpacker.lambda$loadFile$1(Unpacker.java:174)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
```

**Comment by LordMonoxide** [comment]
Fixed by #1231
